vendor/vendor.json

```diff
@@ -8,6 +8,42 @@
 			"revision": "37407e5ac3ade4d754a2b4eb3f03ac5aefb72cda",
 			"revisionTime": "2020-03-05T11:08:26Z"
 		},
+		{
```

Comment:
I think there is something odd happening with this file. There are all these changes to the vendor stuff but no vendor files in this pull request. Try running `govendor remove +unused` and see what happens